src/NumericalAlgorithms/Interpolation/Callbacks/ObserveSurfaceIntegrals.hpp

```diff
@@ -33,33 +38,79 @@ namespace db {
 template <typename TagsList>
 class DataBox;
 }  // namespace db
+namespace observers {
+namespace ThreadedActions {
+struct WriteReductionData;
+}  // namespace ThreadedActions
+template <class Metavariables>
+struct ObserverWriter;
+}  // namespace observers
 /// \endcond
 
 namespace intrp {
 namespace callbacks {
 
+/// \cond
+namespace detail {
+template <typename T>
+struct reduction_data_type;
+
+template <template <typename...> class T, typename... Ts>
+struct reduction_data_type<T<Ts...>> {
```

Comment:
if this will always be a `tmpl::list`, you don't need the `T` template parameter.